### PR TITLE
chore: add .asyncapi-tool file

### PR DESCRIPTION
**Pull request by TheEques**
Added ,asyncapi-tool file

### CODE REVIEW

File: .asyncapi-tool [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "title": "asyncapi/nodejs-template",
```
Comment (derberg) [inline .asyncapi-tool:2]:
```suggestion
    "title": "Node.js Multiprotocol Template",
```

File: .asyncapi-tool [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    "description": "Generates Node.js service that uses Hermes package",
+    "links": {
+        "repoUrl": "https://github.com/asyncapi/nodejs-template"
```
Comment (derberg) [inline .asyncapi-tool:5]: not needed, tooling solution will figure that out.

**Review by derberg — CHANGES_REQUESTED** [review]
please use YAML not JSON format

File: .asyncapi-tool [hunk 3]
```diff
@@ -0,0 +1,9 @@
+title: Node.js Multiprotocol Template
+description: This template generates a server using your AsyncAPI document. It supports multiple different protocols, like Kafka or MQTT. It is designed in the way that generated code is a library and with it's API you can start the server, send messages or register a middleware for listening incoming messages. Runtime message validation included.
+filters:
+  language: javascript
+  technology:
+  - nodejs
```
Comment (derberg) [inline .asyncapi-tool:6]: I think correct is what you had initially, `Node.js` -> https://github.com/asyncapi/website/blob/master/scripts/tools/tags-color.js

**Review by derberg — COMMENTED** [review]
@akshatnema anything missing?

File: .asyncapi-tool (hunk 3 above)
Comment (derberg) [reply .asyncapi-tool]: you changed to `Nodejs` and was suppose to be `Node.js`

Comment (TheEques) [reply .asyncapi-tool]: I am really sorry didn't see that

**Review by derberg — APPROVED** [review]
Thanks!

**Comment by TheEques** [comment]
@derberg Thanks for all the help tho. I made many minor mistakes this time. I'll make sure not to repeat them in future PRs

**Comment by derberg** [comment]
no worries, you're doing great!

**Pull request merged**